doc/samples/variants/dut.rst

```diff
@@ -0,0 +1,44 @@
+.. _sidewalk_dut:
+
+Device Under Test
+#################
+
+This sample demonstrates a Device Under Test application.
+
+Overview
+********
+
+Device under test generic application having activated CLI including functionality for on_dev_certification for manufacturing process.
+
+.. _sidewalk_dut_user_interface:
+
+User Interface
+**************
+
+No specific buttons and leds actions for dut.
```

Comment:
Buttons and LEDs working like in hello app. Why not describe it also here

Reply:
The last requirement was to prepare a dut sample variant next to hello and demo. It doesn't use any led or buttons